src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +49,10 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> String.valueOf(p.getSex()) == "MAN"
```

Comment:
```suggestion
                .filter(p -> p.getSex() == Person.Sex.MAN
```
we need to compare Strings through `equals` not `==`, and it`s better to use enum here

Reply:
done